xml/System.Text.Json/JsonProperty.xml

```diff
@@ -63,10 +63,23 @@
         <Parameter Name="utf8Text" Type="System.ReadOnlySpan&lt;System.Byte&gt;" />
       </Parameters>
       <Docs>
-        <param name="utf8Text">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="utf8Text">The UTF-8 encoded text to compare against.</param>
+        <summary>Compares the text represented by <paramref name="utf8Text" /> to the name of this property.</summary>
+        <returns><see langword="true" /> if the name of this property has the same UTF-8 encoding as <paramref name="utf8Text" />, <see langword="false" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the name of this property has the same UTF-8 encoding as <paramref name="utf8Text" />; otherwise, <see langword="false" />.</returns>
```